### ISSUE TITLE
FQDN members break deploy in 3.16.0

### ISSUE THREAD

**Issue by zivvel** [F5Networks/f5-appsvcs-extension#204]
### Environment
 * Application Services Version: 3.16.0
 * BIG-IP Version:
```
Main Package
  Product     BIG-IP
  Version     13.1.3
  Build       0.21.6
  Edition     Engineering Hotfix
  Date        Tue Aug 20 12:51:19 PDT 2019

Hotfix List
ID657912-3    ID778517-2
```

### Summary

Since upgrading AS3 from 3.13.1 to 3.16.0 (THANK YOU for dots and dashes in object names!), when I post seemingly anything that modifies an existing tenant/app that uses FQDN pool members, I get an error:

0107189b:3: Cannot delete ephemeral object: /tenant/app/pool_name: /tenant/poolmemberFQDN.

If I remove/deploy, it always works.

I am not using shareNodes

### Steps to Reproduce:

1. POST Declaration (pool member must resolve to at least one IP):
```json
{
  "class": "AS3",
  "action": "deploy",
  "persist": true,
  "declaration": {
    "class": "ADC",
    "schemaVersion": "3.16.0",
    "id": "test-0001",
    "test": {
      "class": "Tenant",
      "test_w11": {
        "class": "Application",
        "template": "generic",
        "testapp_80": {
          "class": "Service_TCP",
          "virtualAddresses": [
            "10.255.254.12"
          ],
          "virtualPort": 80,
          "pool": "testapp_pool",
        },
        "testapp_pool": {
          "class": "Pool",
          "members": [{
            "servicePort": 80,
            "addressDiscovery": "fqdn",
            "autoPopulate": true,
            "hostname": "www.f5.com",
          }]
        },
      }
    }
  }
}
```

2. POST Modified Declaration.
In this case, I added a 2nd Service_TCP on port 81.  This happens with seemingly _any_ modification.
```json
{
  "class": "AS3",
  "action": "deploy",
  "persist": true,
  "declaration": {
    "class": "ADC",
    "schemaVersion": "3.16.0",
    "id": "test-0001",
    "test": {
      "class": "Tenant",
      "test_w11": {
        "class": "Application",
        "template": "generic",
        "testapp_80": {
          "class": "Service_TCP",
          "virtualAddresses": [
            "10.255.254.12"
          ],
          "virtualPort": 80,
          "pool": "testapp_pool",
        },
        "testapp_81": {
          "class": "Service_TCP",
          "virtualAddresses": [
            "10.255.254.12"
          ],
          "virtualPort": 81,
          "pool": "testapp_pool",
        },
        "testapp_pool": {
          "class": "Pool",
          "members": [{
            "servicePort": 80,
            "addressDiscovery": "fqdn",
            "autoPopulate": true,
            "hostname": "www.f5.com",
          }]
        },
      }
    }
  }
}
```

3. Observe the following error response:
```
{
  "changed": false,
  "msg": "0107189b:3: Cannot delete ephemeral object: /test/test_w11/testapp_pool: /test/www.f5.com."
}
```

Edit: typo in BIG-IP Version: 13, not 12!
Edit: add steps to reproduce.

**Comment by zivvel** [comment]
Opened case C3147615 with F5 Support for this.

**Comment by dstokesf5** [comment]
In the internal product backlog as AUTOTOOL-1039.

**Comment by dstokesf5** [comment]
This should be fixed in the 3.17.0 release. If you are still experiencing a problem, please re-open this issue or create a new one.
